webapp.py

```
@@ -283,5 +292,46 @@ def set_runes_preference():
         )
 
 
+@api.route("/actions/sendrunes", methods=["POST"])
+@ensure_connection
+def set_runes():
+    try:
+        runes.send_runes_and_summs(state.connection)
+        return empty_success_response()
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)